src/presentation/ui/component/form/selectInput.templ

```diff
@@ -88,3 +88,80 @@ templ SelectInputWithLabelValuePair(
 		<i class="ph-bold ph-caret-down pointer-events-none absolute right-2.5 top-3.5 text-sm"></i>
 	</div>
 }
+
+script MultipleSelectInputLocalState() {
+	document.addEventListener('alpine:init', () => {
+		Alpine.data('multipleSelectInput', () => ({
+			getFormattedSelectedItems(bindSelectedItemsPath) {
+				if (bindSelectedItemsPath.length == 0) {
+					return '--';
+				}
+
+				let formattedSelectedItems = bindSelectedItemsPath.join(', ');
+				if (formattedSelectedItems.length > 80) {
+					return formattedSelectedItems.substr(0, 80) + '...';
+				}
+				return formattedSelectedItems;
+			},
+			isOptionsExpanded: false,
+			closeDropdown() {
+				this.$refs.multipleSelectInputOptions.classList.add('hidden');
```

Comment:
Why this instead of using x-show on line 148?

Reply:
Done!